packages/chatbot-server-mongodb-public/src/verifiedAnswers.eval.ts

```diff
@@ -0,0 +1,291 @@
+  fs.readFileSync(verifiedAnswersPath, "utf-8")
+);
+const verifiedAnswerIndex = makeVerifiedAnswerIndex(verifiedAnswerSpecs);
+
+const verifiedAnswerEvalCases: VerifiedAnswersEvalCase[] = [
+  makeVerifiedAnswerEvalCase({
+    inputQuery: "what is the aggregation framework",
+    similarVerifiedAnswerQuery: "What is aggregation in MongoDB?",
+    tags: ["aggregation", "perturbation"],
+    verifiedAnswerIndex,
+  }),
+  makeVerifiedAnswerEvalCase({
+    inputQuery: "agg framework",
+    similarVerifiedAnswerQuery: "What is aggregation in MongoDB?",
+    tags: ["aggregation", "perturbation", "should_match"],
+    verifiedAnswerIndex,
+  }),
+  makeVerifiedAnswerEvalCase({
+    inputQuery: "what's the process to insert data into MongoDB",
+    similarVerifiedAnswerQuery: "How do I insert data into MongoDB?",
+    verifiedAnswerIndex,
+    tags: ["perturbation", "should_match"],
+  }),
+  makeVerifiedAnswerEvalCase({
+    inputQuery: "How can I insert data into MongoDB?",
+    similarVerifiedAnswerQuery: "How do I insert data into MongoDB?",
+    verifiedAnswerIndex,
+    tags: ["perturbation", "should_match"],
+  }),
+  makeVerifiedAnswerEvalCase({
+    inputQuery: "insert data into mongodb",
+    similarVerifiedAnswerQuery: "How do I insert data into MongoDB?",
+    verifiedAnswerIndex,
+    tags: ["perturbation", "should_match"],
+  }),
+  makeVerifiedAnswerEvalCase({
+    inputQuery: "password reset",
+    similarVerifiedAnswerQuery: "Can i reset my password",
+    verifiedAnswerIndex,
+    tags: ["perturbation", "iam", "should_match"],
+  }),
+  makeVerifiedAnswerEvalCase({
+    inputQuery: "reset my password",
+    similarVerifiedAnswerQuery: "Can i reset my password",
+    verifiedAnswerIndex,
+    tags: ["perturbation", "iam", "should_match"],
+  }),
+  makeVerifiedAnswerEvalCase({
+    inputQuery: "reset database password",
+    similarVerifiedAnswerQuery: "Can i reset my password",
+    verifiedAnswerIndex,
+    tags: ["perturbation", "iam", "should_match"],
+  }),
+  makeVerifiedAnswerEvalCase({
+    inputQuery: "connect to stream process",
+    verifiedAnswerIndex,
+    tags: ["atlas_stream_processing", "should_not_match"],
+  }),
+  makeVerifiedAnswerEvalCase({
+    inputQuery: "connect to database kotlin",
+    verifiedAnswerIndex,
+    tags: ["driver", "kotlin", "should_not_match"],
+  }),
+  makeVerifiedAnswerEvalCase({
+    inputQuery: "connect to database with Kotlin coroutine driver",
+    verifiedAnswerIndex,
+    tags: ["driver", "kotlin", "kotlin_coroutine_driver", "should_not_match"],
+  }),
+  // 👇 From EAI-580 👇
+  makeVerifiedAnswerEvalCase({
+    inputQuery: "how do I set up billing alerts in Atlas",
+    // No similar verified answer
+    tags: ["billing", "should_not_match"],
+    verifiedAnswerIndex,
+  }),
+  makeVerifiedAnswerEvalCase({
+    inputQuery: "how do I set up billing alerts in Atlas?",
+    // No similar verified answer
+    tags: ["billing", "should_not_match"],
+    verifiedAnswerIndex,
+  }),
```

Comment:
```suggestion
```

Is this an unintended dupe of the previous entry?

Reply:
yea looks like it, nice catch